tests/test_codetiming.py

```diff
@@ -164,3 +166,13 @@ def test_error_if_restarting_running_timer():
     t.start()
     with pytest.raises(TimerError):
         t.start()
+
+
+def test_timer_sets_last():
+    t = Timer()
+    assert isnan(t.last)
+    t.start()
+    time.sleep(0.1)
+    t.stop()
+
+    assert 0.1 < t.last < 1
```

Comment:
While I'm sure this should always pass the test, I think that `time.sleep()` only guarantees that we can `assert t.last >= 0.1`? And I don't think testing the upper limit gives us anything useful?

Reply:
yep, that's true!